build.gradle

```diff
@@ -149,6 +153,19 @@ subprojects {
       exceptionFormat "full"
     }
   }
+
```

Comment:
placing this entire block under `subprojects` right after the `test` block in https://github.com/apache/iceberg/blob/b4ed29b2af16aa88b5da842c64a8cc3630583ada/build.gradle#L151 worked for me